test.jl

```diff
@@ -0,0 +1,19 @@
+include("src/RedisTimeSeries.jl")
+import .RedisTimeSeries
+import Redis
+
+conn = Redis.RedisConnection() # host=127.0.0.1, port=6379, db=0, no password
+
+key = "temperature:2:32"
+Redis.del(conn, key)
+RedisTimeSeries.create(conn, key, "RETENTION 60000 LABELS sensor_id 2 area_id 32")
+RedisTimeSeries.alter(conn, key, "LABELS sensor_id 2 area_id 32 sub_area_id 15")
+RedisTimeSeries.add(conn, key, 1548149180000, 26.0, Dict{String,String}("sensor_id" => "2", "area_id" => "32"))
```

Comment:
@jeffreylovitz can you please convert it to a real test with assert so we can use is as a base for our future tests?